usage.py

```diff
@@ -1,23 +1,61 @@
-from realtime.connection import Socket
+import asyncio
+from typing import Optional
 
+from realtime.channel import RealtimeSubscribeStates
+from realtime.client import RealtimeClient
 
-def callback1(payload):
-    print("Callback 1: ", payload)
 
+async def main():
+    REALTIME_URL = "ws://localhost:4000/websocket"
+    API_KEY = "1234567890"
 
-def callback2(payload):
-    print("Callback 2: ", payload)
+    socket = RealtimeClient(REALTIME_URL, API_KEY)
```

Comment:
how will this look when we use in `supabase-py` ?

Reply:
just like js, SupabaseClient gives access to the realtime client instance, but also exposes convenience methods for managing channels, example:

```python
supabase = create_client(...)

supabase.realtime # for full access

channel = supabase.channel("topic", params={"config": {"broadcast": {"self": True}}})
all_channels = supabase.get_all_channels()
```

We still need to implement this on `supabase-py`, but that is the ideia, if you have any recommendation, it is welcomed.

Reply:
perfect ![image](https://github.com/user-attachments/assets/85bcc256-e82f-400f-add4-26627983c103)